bool is_activated
int8 is_episode_done